.github/labeler.yml

```diff
@@ -0,0 +1,6 @@
+docs:
+- "**/*.md"
+exercise:
+- "exercises/**"
+config change:
+- "config.json"
```

Comment:
This should be in a different PR.

You can add a commit deleting it and make a new PR including it :)